### DISCUSSION TITLE
部署在Vercel ，但是hexo在服务器上，能关联上吗？

### DISCUSSION THREAD

**Discussion by hyy34** [Qexo/Qexo#447] [Q&A]
我看后台有本地的功能，但是这个好像是部署在本地才可以，我本地死活部署不上。。@@

**Comment by am-abudu** [comment]
不行。我有考虑过写Ftp驱动器，可惜时间不允许。你可以考虑把Qexo部署到服务器上

**Reply by am-abudu** [reply]
如果有能力的话欢迎PR，FTP驱动器。应该不算麻烦
